exp/views/lab.py

```diff
@@ -411,6 +411,7 @@ def notify_admins_of_lab_submission(self):
                     name=SiteAdminGroup.LOOKIT_ADMIN.name
                 ).user_set.values_list("username", flat=True)
             ),
+            reply_to=[self.request.user.username],
```

Comment:
nice job finding both of these, I hadn't realized they were buried in somewhat different places!